doc/docs/reference/core_lookuplists.md

```diff
@@ -16,3 +16,26 @@ for l in lookuplists.lookuplists():
 
 # -> Drug, Gender, Condition et cetera
 ```
+
+
+## {{ LookupListCls }}.objects.search(some_str, case_sensitive=False, contains=False)
+A manager method that takes a string and will search accross the lookup list itself or synonyms. If `contains=True` it will look for partial matches. By default it is case
+insensitive, to change this behaviour query with `case_sensitive=True`
+
+```python
+from opal.core import Antimicrobial
+
+Antimicrobial.objects.search("Difficulty Swallowing")
+## returns the Antimicrobial Dysphagia
```

Comment:
I'm not medically qualified, but this doesn't look like an antimicrobial to me... can we have an example that is actually the thing that the example is showing ? ;D